### DISCUSSION TITLE
Not possible to combine Goal Conversions breakdowns

### DISCUSSION THREAD

**Discussion by kleinicke** [plausible/analytics#981] [Anything Else]
In the Goal Conversions, the goals can be separated into several categories. Right now it is only possible to choose one selection in one of the breakdown categories. 
![Breakdown](https://user-images.githubusercontent.com/10370975/116785868-20645b00-aa9c-11eb-9e17-c69451e0af4e.png)

In the example in the graphic above it would not be possible to choose "Student" from the "type" category and additionally a value from the "chapter" category. We helped ourselves by creating a combined "chapter+type" category, but it would be easier if this wouldn't be necessary.

**Comment by ukutaht** [comment]
Thanks for reporting!

This is a known limitation with the database we're using: https://github.com/plausible/analytics/discussions/913#discussioncomment-580351

It will be fixed soon-ish, the team behind the database is testing a better way to store an arbitrary mapping of values.
